### PR TITLE
input's info tooltips added by passing shinypanels' infoTooltip funct…

### LINKED ISSUE #26: add possibility of adding informative tooltips in inputs
```{r}
library(shinypanels)

styles <- "
.info-tool {
 display: flex;
}


.tooltip-inf {
 cursor: pointer;
 position: relative;
 margin-left: 3px;
}

.tooltip-inf .tooltiptext {
  visibility: hidden;
  width: 170px;
  higth: auto;
  background-color: #fafafa;
  color: #9FBD36;
  position: absolute;
  z-index: 9999;
  top: 0;
  padding: 1rem;
  border: 1px solid #ccc;
  font-weight: 400;
  letter-spacing: normal;
  font-size: 0.75rem;
}

.tooltip-inf:hover .tooltiptext {
  visibility: visible;
}
"

ui <- panelsPage( styles = styles,
                  panel(title = "example", 
                        head = NULL,
                        body = uiOutput('ejemplo')
                  )
)

server <- function(input, output, session) {
  
  output$ejemplo <- renderUI({
    div(
    HTML('<div class= "info-tool"> Plot type  <div class="tooltip-inf"> 
    <i class="fa fa-info-circle"></i><span class="tooltiptext">
    Its a tootip information </span</div></div></div>'),
    radioButtons('plot_type',  ' ', c("ggplot", "hgchmagic")))
})

}

shinyApp(ui, server)
```

In Yaml one option with:
---
library:
  show: true
  tooltip_info: Its a tootip information
  input_type: radioButtons
  input_params:
    label: Library
    choices:
      highcharter:
      - Interactive
      ggplot:
      - Static
    inline: true

### PR DESCRIPTION
**Pull request by brazadas**
…ion to label argument in every widget; also test added closes #26

### CODE REVIEW

**Comment by jpmarindiaz** [comment]
Tests passing... OK.
But the app in EX01, doesn't work for me, any ideas?

image

File: inst/examples/ex01-basic/app.R [hunk 1]
```diff
@@ -1,7 +1,35 @@
 library(shiny)
 library(parmesan)
 
-ui <- fluidPage(
+styles <- "
```
Comment (jpmarindiaz) [inline inst/examples/ex01-basic/app.R]: Please include all styles directly in the package

File: inst/examples/ex01-basic/app.R [hunk 2]
```diff
@@ -17,8 +45,10 @@ ui <- fluidPage(
 
 server <-  function(input, output, session) {
 
-  path <- system.file("examples", "ex01-basic", "parmesan", package = "parmesan")
-  parmesan <- parmesan_load(path)
+  # path <- system.file("examples", "ex01-basic", "parmesan", package = "parmesan")
+  # parmesan <- parmesan_load(path)
+  parmesan <- parmesan_load("parmesan")
```
Comment (jpmarindiaz) [inline inst/examples/ex01-basic/app.R]: No need to remove how we load files in all examples. Use devtools::load_all of crtl + shift + L to test locally

File: inst/examples/ex01-basic/parmesan/inputs.yaml [hunk 3]
```diff
@@ -7,6 +7,9 @@ bins:
     max: 50
     value: 10
     step: 1
+  input_info:
+    icon:
```
Comment (jpmarindiaz) [inline inst/examples/ex01-basic/parmesan/inputs.yaml]: do not leave blank, even when you are providing a default when NULL users will use the examples to learn how to change it. Use icon: info-circle

File: inst/examples/ex01-basic/parmesan/inputs.yaml [hunk 4]
```diff
@@ -7,6 +7,9 @@ bins:
...
+    text: fjsdksjdkskds
```
Comment (jpmarindiaz) [inline inst/examples/ex01-basic/parmesan/inputs.yaml]: Use informative texts for all examples

**Review by jpmarindiaz — CHANGES_REQUESTED** [review]
Fix comments and submit again. Example 01 should work even with shinypanels not loaded. Either import the shinypanels::infoTooltip function in the docs or duplicate it locally in this package.
Do not pollute example code with unnecesary CSS styles. If they are needed, they should go inside the package.

**Comment by brazadas** [comment]
Listo!

**Pull request merged**